mypy/messages.py

```diff
@@ -520,6 +522,14 @@ def does_not_return_value(self, void_type: Type, context: Context) -> None:
             self.fail('{} does not return a value'.format(
                 capitalize((cast(Void, void_type)).source)), context)
 
+    def deleted_as_rvalue(self, typ: DeletedType, context: Context) -> None:
+        """Report an error about using an deleted type as an rvalue."""
+        if typ.source is None:
+            s = ""
+        else:
+            s = " '{}'".format(typ.source)
+        self.fail('Using deleted variable{} as an rvalue'.format(s), context)
```

Comment:
The message might be a little confusing for people not familiar with PL terminology. What about rewording the message as `Can't read deleted variable 'foo'`, for example?